### PR TITLE
Add `rad-auth`, `rad-help` commands

### PR DESCRIPTION
**Pull request by xphoniex**

### CODE REVIEW

File: radicle-cli/src/commands/auth.rs [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    --active                Authenticate with the currently active profile
+    --stdin                 Read passphrase from stdin (default: false)
+    --name <name>           Use given name (default: none)
+    --help                  Print help
+"#,
+};
+
+#[derive(Debug)]
+pub struct Options {
+    pub active: bool,
```
Comment (cloudhead) [inline radicle-cli/src/commands/auth.rs:41]: Can be removed as well. `rad auth` run when the user has a profile is `rad-auth --active`

File: radicle-cli/src/commands/auth.rs [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+        term::success!("Signing key already in ssh-agent");
+    };
+
+    Ok(())
+}
+
+#[cfg(test)]
+mod tests {
+    use anyhow::Result;
+    use assay::assay;
```
Comment (cloudhead) [inline radicle-cli/src/commands/auth.rs:191]: Let's get rid of `assay`

File: radicle-cli/src/common.rs [hunk 3]
```diff
@@ -0,0 +1,88 @@
+pub mod git {
```
Comment (cloudhead) [inline radicle-cli/src/common.rs:1]: Delete in favor of `src/git.rs`

File: radicle-cli/src/commands/auth.rs (hunk 2 above)
Comment (xphoniex) [reply radicle-cli/src/commands/auth.rs:191]: should I entirely remove tests for now? since it depends on original `common`

Comment (cloudhead) [reply radicle-cli/src/commands/auth.rs:191]: Yup

**Comment by cloudhead** [comment]
Needs rebase, as I added `rad-checkout` as well to the other PR.

File: radicle-cli/src/commands/auth.rs [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+    A passphrase may be given via the environment variable `RAD_PASSPHRASE` or
+    via the standard input stream if `--stdin` is used. Using one of these
+    methods disables the passphrase prompt.
+
+    If `--init` is used, a name may be given via the `--name` option. Using
+    this disables the input prompt.
+
+Options
+
+    --init                  Initialize a new identity
```
[the remark this answers is not in the document]
Comment (xphoniex) [reply radicle-cli/src/commands/auth.rs]: Also removed `name` since identity is only a key now.

File: radicle-cli/src/commands/auth.rs [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+Usage
+
+    rad auth [<options>...]
+
+    A passphrase may be given via the environment variable `RAD_PASSPHRASE` or
+    via the standard input stream if `--stdin` is used. Using one of these
+    methods disables the passphrase prompt.
+
+    If `--init` is used, a name may be given via the `--name` option. Using
+    this disables the input prompt.
```
Comment (cloudhead) [inline radicle-cli/src/commands/auth.rs:27]: This paragraph can be removed

File: radicle-cli/src/commands/auth.rs [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+    }
+}
+
+pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
+    let profile = Profile::load();
+
+    if profile.is_err() {
+        init(options)
+    } else {
+        authenticate(options, ctx)
```
Comment (cloudhead) [inline radicle-cli/src/commands/auth.rs:70]: Profile should be passed here.

File: radicle-cli/src/commands/auth.rs [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+                _ => return Err(anyhow::anyhow!(arg.unexpected())),
+            }
+        }
+
+        Ok((Options { stdin }, vec![]))
+    }
+}
+
+pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
+    let profile = Profile::load();
```
Comment (cloudhead) [inline radicle-cli/src/commands/auth.rs:65]: `ctx.profile()?` ?

File: radicle-cli/src/commands/help.rs [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+pub use rad_path;
+pub use rad_pull;
+pub use rad_push;
+pub use rad_remote;
+pub use rad_review;
+pub use rad_rm;
+pub use rad_sync;
+pub use rad_track;
+pub use rad_untrack;
+*/
```
Comment (cloudhead) [inline radicle-cli/src/commands/help.rs:36]: Remove

File: radicle-cli/src/commands/help.rs [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+    rad_track::HELP,
+    rad_untrack::HELP,
+    rad_sync::HELP,
+    #[cfg(feature = "ethereum")]
+    rad_ens::HELP,
+    #[cfg(feature = "ethereum")]
+    rad_account::HELP,
+    rad_rm::HELP,
+    rad_edit::HELP,
+    */
```
Comment (cloudhead) [inline radicle-cli/src/commands/help.rs:66]: Remove

File: radicle-cli/src/main.rs [hunk 10]
```diff
@@ -0,0 +1,343 @@
...
+        }
+        "untrack" => {
+            term::run_command_args::<rad_untrack::Options, _>(
+                rad_untrack::HELP,
+                "Untrack",
+                rad_untrack::run,
+                args.to_vec(),
+            );
+        }
+        */
```
Comment (cloudhead) [inline radicle-cli/src/main.rs:321]: Remove

File: radicle-cli/src/commands/auth.rs [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
+    let profile = ctx.profile();
+
+    if profile.is_err() {
+        init(options)
+    } else {
+        authenticate(&profile?, options, ctx)
+    }
```
Comment (cloudhead) [inline radicle-cli/src/commands/auth.rs:68]: This can now be turned into a match statement to avoid the `?`

**Pull request merged**